test/suites/storage_driver_ceph.sh

```diff
@@ -40,6 +40,22 @@ test_storage_driver_ceph() {
     lxc storage create "lxdtest-$(basename "${LXD_DIR}")-valid-ceph-pool-config" ceph volume.block.filesystem=ext4 volume.block.mount_options=discard volume.size=25MiB ceph.rbd.clone_copy=true ceph.osd.pg_num=16
     lxc storage delete "lxdtest-$(basename "${LXD_DIR}")-valid-ceph-pool-config"
 
+    # Test that if there is a temporay error on the Ceph side (network jitter, wrong auth key, etc.)
+    # while creating a storage pool that already exists, lxd will not delete this storage pool.
+    # Step 1: Create a new Ceph storage pool
+    lxc storage create "lxdtest-$(basename "${LXD_DIR}")-pool3" ceph volume.size=25MiB ceph.osd.pg_num=16
+    # Step 2: Change the Ceph admin client key to a wrong value in /var/snap/microceph/current/conf/ceph.keyring
+    original_key=$(awk -F "key = " '/key/ {print $2}' /var/snap/microceph/current/conf/ceph.keyring)
```

Comment:
Sorry, we cannot have the test suite depend on microceph, as the test suite is used by non-ubuntu and non-microceph users.

Reply:
I'm not sure we can test this well in the test suite as we do not control the ceph environment.

Reply:
Yes, I was wondering how to do this cleanly but I clearly ran out of ideas...

Reply:
lets take the test parts that depend on microceph out for now.

Reply:
If we are taking them out, then I don't see the point of this test at all as the main purpose is to simulate some kind of Ceph outage.. shall we just remove this test commit ?

Reply:
yep ok with me